### ISSUE TITLE
Issue loading pill

### ISSUE THREAD

**Issue by crrncllns** [urbit/support#9]
https://twitter.com/intocryptoast/status/1255989174156447744

**Comment by vvisigoth** [comment]
@joemfb have you seen anything like this before? I don't know why the pill would download but then not "load."

**Comment by joemfb** [comment]
No, that's very strange. It'd be helpful to see the full command and console output.

**Comment by vvisigoth** [comment]
To update this: he can curl the pill file, so something is borked in the loading of the pill.

**Comment by vvisigoth** [comment]
@rmariani can you look into this? it seems that our WSL docs are off somehow, since that's where he's trying to deal with this. I'll also forward you the email thread.

**Comment by vvisigoth** [comment]
closed the wrong issue, apologies.

**Comment by vvisigoth** [comment]
For posterity, user is trying to update to WSL2 from WSL1.

**Comment by botter-nidnul** [comment]
For future reference, because starting in about 3 weeks there will be a stream of people who will be confused as to why they can't run Urbit even though they upgraded to WSL2:

Just because Windows upgraded from WSL1 to WSL2 doesn't mean your distro upgraded from WSL1 to WSL2.

In a windows console run:
`wsl --list --verbose` to discover the exact name of your distro to use in the next command
`wsl --set-version <Distro> 2`

And to avoid distros being created with the wrong version in the future:
`wsl --set-default-version 2`

**Comment by vvisigoth** [comment]
oh wow, thanks @botter-nidnul  !

**Comment by rmariani** [comment]
> For posterity, user is trying to update to WSL2 from WSL1.

Did he manage to do so?
